README.md

```diff
@@ -629,6 +629,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Pippo](http://www.pippo.ro/) - Small, highly modularized Sinatra-like framework.
 * [Play](https://www.playframework.com/) - Uses convention over configuration, hot code reloading and display of errors in the browser.
 * [PrimeFaces](http://primefaces.org/) - JSF framework which has a free and a commercial version with support. Provides several frontend components.
+* [Rapidoid](http://www.rapidoid.org/) - An Open source (Apache license), Simple, Secure and Extremely fast Web Framework for Java.
```

Comment:
The consensus was to write `open-source`, also you don't need to mention the license. Also the description is very much generic, could you please fix it and be as concise as possible?